onebusaway-api-webapp/src/main/resources/data-sources.xml

```diff
@@ -86,4 +86,12 @@
             </props>
         </property>
     </bean>
+    <bean id="customRouteSort" class="org.onebusaway.transit_data.model.RouteSort">
```

Comment:
Question:  does customRouteSort always need to be defined?  Should there be an emptyRouteSort that's an no-op?

Reply:
The compareRoutes method in RouteSort class will check whether configuration is provided and if not it will use  a standard sort so no need of an emptyRouteSort with no-op.